platform-core-module/src/main/java/com/softicar/platform/core/module/maintenance/session/SessionOverviewPageDiv.java

```diff
@@ -1,12 +1,53 @@
 package com.softicar.platform.core.module.maintenance.session;
 
+import com.softicar.platform.core.module.CoreI18n;
+import com.softicar.platform.core.module.CoreImages;
 import com.softicar.platform.dom.elements.DomDiv;
+import com.softicar.platform.dom.elements.bar.DomActionBar;
+import com.softicar.platform.dom.elements.button.DomButton;
 import com.softicar.platform.emf.data.table.EmfDataTableDivBuilder;
+import com.softicar.platform.emf.data.table.IEmfDataTableActionCell;
+import com.softicar.platform.emf.data.table.IEmfDataTableActionColumnHandler;
+import com.softicar.platform.emf.data.table.IEmfDataTableDiv;
+import javax.servlet.http.HttpSession;
 
 public class SessionOverviewPageDiv extends DomDiv {
 
+	private final IEmfDataTableDiv<HttpSession> table;
+
 	public SessionOverviewPageDiv() {
 
-		new EmfDataTableDivBuilder<>(new SessionOverviewTable()).buildAndAppendTo(this);
+		appendChild(
+			new DomActionBar(
+				new DomButton()//
+					.setClickCallback(new SessionManager()::invalidateAllSessions)
+					.setLabel(CoreI18n.LOGOUT_ALL)
+					.setIcon(CoreImages.LOGOUT.getResource())
```

Comment:
I think we don't need an icon here (much like in the screenshot which seems to be outdated).